slick.grid.js

```diff
@@ -2765,6 +2784,31 @@ if (typeof Slick === "undefined") {
       }
     }
 
+    function setAllColumns(columnDefinitions){
+      var _columns = columns.slice(0);
```

Comment:
if you just want to make a copy, you can omit the 0 and just go with `.slice()`

Reply:
sure, will change - waiting for our conversation to reach an agreement